Migrate EthHub Info & Remove EthHub Links on the state channels developer docs (#9199)
ethereum/ethereum-org-website: src/content/developers/docs/scaling/state-channels/index.md

## Payment channels {#payment-channels}

A payment channel is best described as a "two-way ledger" collectively maintained by two users. The ledger's initial balance is the sum of deposits locked into the on-chain contract during the channel opening phase. Payment channel transfers can be performed instantaneously and without the involvement of the actual blockchain itself, except for an initial one-time on-chain creation and an eventual closing of the channel.

Updates to the ledger's balance (i.e., the payment channel's state) require the approval of all parties in the channel. A channel update, signed by all channel participants, is considered finalized, much like a transaction on Ethereum.

> Payment channels were among the earliest scaling solutions designed to minimize expensive on-chain activity of simple user interactions (e.g., ETH transfers, atomic swaps, micropayments). Channel participants can conduct an unlimited amount of instant, feeless transactions between each other as long as the net sum of their transfers does not exceed the deposited tokens. [Raiden](https://linktr.ee/RaidenNetwork) is an example of a payment channel on Ethereum.

Reviewer: We should avoid calling out (advertising) specific solutions when explaining concepts.

```suggestion
Payment channels were among the earliest scaling solutions designed to minimize expensive on-chain activity of simple user interactions (e.g. ETH transfers, atomic swaps, micropayments). Channel participants can conduct an unlimited amount of instant, feeless transactions between each other as long as the net sum of their transfers does not exceed the deposited tokens.
```

## State channels {#state-channels}

Apart from supporting off-chain payments, payment channels have not proven useful for handling general state transition logic. State channels were created to solve this problem and make channels useful for scaling general-purpose computation.

> State channels still have a lot in common with payment channels. For example, users interact by exchanging cryptographically signed messages (transactions), which the other channel participants must also sign. If a proposed state update isn't signed by all participants, it is considered invalid.

Reviewer: ```suggestion
State channels still have a lot in common with payment channels. For example, users interact by exchanging cryptographically signed messages (transactions), which the other channel participants must also sign. If a proposed state update isn't signed by all participants, it is considered invalid.
```

3. **Latency**: Off-chain transactions conducted between channel participants can be settled instantly, if both parties cooperate, reducing delays. In contrast, sending a transaction on Mainnet requires waiting for nodes to process the transaction, produce a new block with the transaction, and reach consensus. Users may also need to wait for more block confirmations before considering a transaction finalized.

> 4. **Cost**: State channels are particularly useful in situations where a set of participants will be exchanging many state updates over a long period of time. This is because the only cost incurred by the channel is the initial setup and closure of the state channel smart contract. Consequently, every additional intermediary state change in the channel will be cheaper than the last as the settlement cost is distributed accordingly.

Reviewer: ```suggestion
4. **Cost**: State channels are particularly useful in situations where a set of participants will exchange many state updates over a long period. This is because the only cost incurred by the channel is the initial setup and closure of the state channel smart contract. Consequently, every additional intermediary state change in the channel will be cheaper than the last as the settlement cost is distributed accordingly.
```
